example/lib/main.dart

```diff
@@ -1,9 +1,13 @@
 import 'package:flutter/material.dart';
 import 'dart:async';
-
 import 'package:alarm/alarm.dart';
+import 'package:alarm/shared_preferences.dart';
 
-void main() => runApp(const MaterialApp(home: MyApp()));
+void main() {
+  WidgetsFlutterBinding.ensureInitialized();
+  SharedPreference.init();
```

Comment:
Why do you init SharedPreference in the example app ? I think you should do it in the init method of the Alarm service.